src/Table.tsx

```diff
@@ -71,14 +73,16 @@ export function Table<RowShape>({
       </thead>
       <tbody>
         {data.map((item, index) => (
-          <tr>
+          <tr key={index}>
             {columns.map(
               ({
                 render
-              }) => (
+              }, rowIndex) => (
                 <td 
+                  key={rowIndex}
```

Comment:
Same issue here, the `key` shouldn't be the positional index (this index also isn't the `rowIndex` but is rather the `columnIndex`)   

We should require that each `column` specify its own `id` (in addition to `headerTitle` and `render` (the ambiguity of this `render` prop might come back to bite us later if we want to differentiate between render functions for cells and headers))